wrappers/nodejs/src/addon.cpp

```diff
@@ -335,7 +335,7 @@ class RSOptionRange : public DictBase {
 
 class RSNotification : public DictBase {
  public:
-  RSNotification(const char* des, rs2_time_t time, rs2_log_severity severity,
+  RSNotification(std::string des, rs2_time_t time, rs2_log_severity severity,
       rs2_notification_category category, std::string serialized_data) {
```

Comment:
This change exposes an existing problem of using `std::string`, we should use `std::string&` as much as possible, I read the inside code, I do not see any reason we do not go that way.

Reply:
Yes, this has been updated.